src/worker.ts

```diff
@@ -14,8 +26,28 @@ export default {
 			case EVERY_11_HOURS: {
 				return await addMissedUpdatesRole(env);
 			}
-			default:
+
+			case EVERY_20_MINUTES: {
+				await syncIdleUsers(env);
+				await syncNickNames(env);
+				await syncIdle7dUsers(env);
+				await syncOnboarding31dPlusUsers(env);
+				console.log('Worker for syncing idle users, nicknames, idle 7d users, and onboarding 31d+ users has completed.');
+				break;
+			}
+
+			case EVERY_30_MINUTES: {
+				await syncUsersStatus(env);
+				await syncExternalAccounts(env);
+				await syncUnverifiedUsers(env);
+				console.log('Worker for syncing user status, external accounts, and unverified users has completed.');
+				break;
+			}
```

Comment:
Is this the requirement? 
how was the decision of 20 minutes and 30 minutes made?

Reply:
For now we can plan 5 APIs every 30 minutes, totaling 240 API calls over a 24-hour period and 4 APIs every 20 minutes, which would also amount to 288 API for 24 hrs

Our current capacity allows for up to 100,000 API calls per day under the 20- and 30-minute triggers. The reason for not setting triggers less than 20 minutes is due to the existence of /commands in addition to worker processes.
The Decision can be made based on the data of the prod

Reply:
> For now we can plan 5 APIs every 30 minutes, totaling 240 API calls over a 24-hour period and 4 APIs every 20 minutes, which would also amount to 288 API for 24 hrs
> 
> Our current capacity allows for up to 100,000 API calls per day under the 20- and 30-minute triggers. The reason for not setting triggers less than 20 minutes is due to the existence of /commands in addition to worker processes. The Decision can be made based on the data of the prod

can't we make it from configuration file..?